plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -501,14 +500,14 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    std::atomic<int32_t>                              _max_transaction_time_ms; // modified by app thread, read by net_plugin thread pool
    std::atomic<uint32_t>                             _received_block{0};       // modified by net_plugin thread pool
    fc::microseconds                                  _max_irreversible_block_age_us;
-   int32_t                                           _cpu_effort_us = 0;
+   // produce-block-offset is in terms of the complete round, internally use calculated value for each block of round
+   fc::microseconds                                  _produce_block_cpu_effort_us;
```

Comment:
`produce` is not needed in this name.

Reply:
I added `produce` here to emphasize that it controls the block production block effort not subjective block production effort. I can drop `produce` if you think it is clearer.

Reply:
OK, keep produce here.